waiter/src/waiter/websocket.clj

```diff
@@ -297,24 +304,21 @@
         (timers/start-stop-time!
           stream
           (when-let [close-message (async/<! request-close-promise-chan)]
-            (let [[source close-code status-code close-message] close-message]
-              (log/info "websocket connections requested to be closed due to" close-code)
+            (let [[source close-code status-code-or-exception close-message] close-message]
+              (log/info "websocket connections requested to be closed due to" source close-code close-message)
               (counters/dec! requests-streaming)
               ;; explicitly close the client connection if backend triggered the close
-              (when (and (= :instance source) (integer? status-code))
```

Comment:
No, it should still have been logged. This changes handles correctly setting the return code to an server error value.